### PR TITLE
Bugfix: Chromograph_prefixes to json. ref: #2265

### PR DESCRIPTION
**Pull request by mikaell**
Add bug fix. Jinja function `tojson` crashed when given `undefined`. This patch removes call to `tojson` as it is no longer in use.

```
File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/cases/templates/cases/case.html", line 456, in block "scripts"
{{case.chromograph_prefixes|tojson}},
File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/json/init.py", line 376, in tojson_filter
return Markup(htmlsafe_dumps(obj, **kwargs))
File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/json/init.py", line 290, in htmlsafe_dumps
dumps(obj, **kwargs)
File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/json/init.py", line 211, in dumps
rv = _json.dumps(obj, **kwargs)
File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/json/init.py", line 238, in dumps
**kw).encode(obj)
File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/json/encoder.py", line 199, in encode
chunks = self.iterencode(o, _one_shot=True)
File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/json/encoder.py", line 257, in iterencode
return _iterencode(o, 0)
File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/json/init.py", line 100, in default
return _json.JSONEncoder.default(self, o)
File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/json/encoder.py", line 180, in default
o.class.name)
TypeError: Object of type 'Undefined' is not JSON serializable
```

This PR adds a functionality or fixes a bug.
OR
This PR marks a new Scout release. We apply semantic versioning. This is a major/minor/patch release for reasons.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by CR

### CODE REVIEW

**Comment by mikaell** [comment]
Tested this fix on Scout stage and it seems to work. Ready to be deployed.

When testing you shouldn't need to empty your cache, but do it anyway, just in case 😀

File: scout/server/blueprints/cases/static/case_images.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ function add_image_to_individual_panel(individuals, prefixes, institute, case_na
  * Draw RHO call pictures -coverage- and UPD pictures -color coded
  * genome regions- onto the dashboard.
  */
-function draw_tracks(individual, prefixes, institute, case_name){
+function draw_tracks(individual,  institute, case_name){
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/static/case_images.js:56]:
```suggestion
function draw_tracks(individual, institute, case_name){
```

**Review by northwestwitch — APPROVED** [review]
Looks fine, just a suggestion to remove an extra space in the js code. Tested on stage on one of the samples that triggered the error and it works now! 👍

**Comment by northwestwitch** [comment]
fix #2265

**Pull request merged**